ext/mbstring/libmbfl/filters/mbfilter_cp1252.c

```diff
@@ -79,27 +79,25 @@ const struct mbfl_convert_vtbl vtbl_wchar_cp1252 = {
  */
 int mbfl_filt_conv_wchar_cp1252(int c, mbfl_convert_filter *filter)
 {
-	int s=-1, n;
+	int s = -1;
 
 	if (c >= 0x100)	{
+		if (c == 0xFFFD) {
```

Comment:
Why is it necessary to handle 0xFFFD on the wchar -> cp1252 side? I would expect this to only be handled during cp1252 -> wchar, because the table uses it.

Reply:
You are absolutely right. Cleaning that up now.